src/test/system_tests/ceph_s3_tests/test_ceph_s3.js

```diff
@@ -76,8 +75,9 @@ async function run_s3_tests() {
 
 async function run_all_tests() {
     console.info('Running Ceph S3 Tests...');
+    const tox_args = `-c ${CEPH_TEST.test_dir}${CEPH_TEST.s3_test_dir}${CEPH_TEST.tox_config}`;
```

Comment:
It seems that `tox_args` is constant and repeats (inside `run_all_tests` and `run_single_test`), maybe take it and add it to the file `test_ceph_s3_constants.js`?